packages/paste-core/components/button/src/DestructiveButton.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+// import * as PropTypes from 'prop-types';
```

Comment:
Do we need PropTypes on these individual components? If not, we can remove this comment.